dockerfiles/init/modules/che-kubernetes-helm/readme.md

```diff
@@ -32,55 +32,53 @@ The context of the commands below is the directory in which this readme file res
 - Or, you can override default values during installation, using the `--set` flag:
 
   ```bash
-  helm upgrade --install <my-che-installation> --namespace <my-che-namespace> --set global.cheDomain=<my-hostname> --set cheImage=<my-image> ./
+  helm upgrade --install <my-che-installation> --namespace <my-che-namespace> --set global.ingressDomain=<my-hostname> --set cheImage=<my-image> ./
   ```
 
-#### Deployment types
-Currenty, only minikube deployment is supported.
+#### Deployment Options
 
 ##### Single User 
 Only Che will be deployed.
 
   ```bash
-  helm upgrade --install <che-release> --namespace <che-namespace> --set global.cheDomain=<domain> ./
+  helm upgrade --install <che-release-name> --namespace <che-namespace> --set global.ingressDomain=<domain> ./
   ```
   
 ##### Multi User 
 Che, KeyCloak and Postgres will be deployed.
 
   ```bash
-  helm upgrade --install <che-release> --namespace <che-namespace> --set global.multiuser=true --set global.cheDomain=<domain> ./
+  helm upgrade --install <che-release-name> --namespace <che-namespace> -f ./values/multi-user.yaml --set global.ingressDomain=<domain> ./
   ```
-  
-##### No Host:
- Ingress will serve requests on minikube-ip. 
- Path based routing to Che, Secondary servers (KeyCloak) and Workspace servers.
 
+#### Default Host
```

Comment:
Shouldn't `Default Host` be at the same level than `Single User` and `Multi User` (a section of `Deployment Options`)? Same for `TLS Enabled`.

Reply:
done